plenum/test/pool_transactions/conftest.py

```diff
@@ -72,6 +73,35 @@ def nodeThetaAdded(looper, txnPoolNodeSet, tdir, client_tdir,
     return newSteward, newStewardWallet, newNode
 
 
+@pytest.fixture(scope='module')
+def sdk_node_theta_added(looper,
+                         txnPoolNodeSet,
+                         tdir,
+                         tconf,
+                         sdk_pool_handle,
+                         sdk_wallet_steward,
+                         allPluginsPath,
+                         testNodeClass=None,
+                         name=None):
+    new_steward_name = "testClientSteward" + randomString(3)
+    new_node_name = name or "Theta"
+    new_steward_wallet, new_node = \
+        sdk_add_new_steward_and_node(looper,
+                                     sdk_pool_handle,
+                                     sdk_wallet_steward,
+                                     new_steward_name,
+                                     new_node_name,
+                                     tdir,
+                                     tconf,
+                                     allPluginsPath,
+                                     nodeClass=testNodeClass)
+    txnPoolNodeSet.append(new_node)
+    looper.run(checkNodesConnected(txnPoolNodeSet))
+    looper.loop.run_until_complete(
+        refresh_pool_ledger(sdk_pool_handle))
```

Comment:
add wrapper func to helper.py, something like sdk_pool_refresh

Reply:
Done